pkg/datapath/iptables/custom_chain.go

```diff
@@ -105,6 +105,18 @@ func (c *customChain) exists(prog iptablesInterface) (bool, error) {
 			return false, nil
 		}
 
+		// with iptables-nft >= 1.8.7, when we try to list the rules of a non existing
```

Comment:
Is there any way to confirm the rule is actually non-existing? just want to make sure that we don't unintentionally skip potentially legitimate errors.

Reply:
I think this is the best option we have at the moment to check if a chain exists :disappointed: 

Looking for some inspiration on the internet I see either this approach being proposed, or a full `iptables-save | grep ..`, which to me seems even less idiomatic